### DISCUSSION TITLE
ESP32 WEMOS: Brownout detector was triggered... Why??

### DISCUSSION THREAD

**Discussion by stefanbode** [arendst/Tasmota#19886] [Q&A] [answered]
Hi, after configuring my ESP32 WEMOS mini I got a brownout reset. I already try different power supply without success. Maybe the device is broken but I have two "new" with the behavior. Any idea? LDO defect??

00:00:00.257 WIF: Attempting connection...

Brownout detector was triggered

ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12736
load:0x40080400,len:2716
entry 0x40080594

The device is "virgin" no connectors, no nothing. Just USB

**Comment by Jason2866** [comment, marked as answer]
The esp32 Wemos Mini do have a to weak on board 3.3V LDO. Same story as for the Wemos D1 mini clones...
Only replacing the LDO against a 500mA type at least will help.

**Reply by sfromis** [reply]
Is it original Wemos, or some bad clone?

**Reply by stefanbode** [reply]
Hi Sfromis. You never know what you get... I got some working and some not but I assume there where from the same order. I hooked it up to an external 3.3V power supply and yes, this works. Therefor the LDO guess was right. Thx

**Reply by stefanbode** [reply]
The 1000microF capacitor do its job. Added this to the 3.3V rail and this fixes the issue like it also does on the old SONOFF devices. Now the LDO can relax...

**Reply by sfromis** [reply]
Some working and some not sounds like bad wannabes with borderline bad specs. There are **no** original Wemos mini ESP32 devices, closest is S2/S3 boards in the D1 Mini form factor. Yeah, I asked a bit of a trick question :grinning: - I know that some sellers happily lie and in the keyword salad title some wannabe board with "Wemos", but that does not make it *original*.

**Reply by stefanbode** [reply]
I really like the form factor and as long as they work I"m fine with a nicely clone. The S2 is my new favorite, but missing for debugging the serial capabilities on the usb. The rest of the limitations is ok for most cases. An alternate to the large capacitor is a parallel LDO, because it is nearly impossible to replace it on the board. This is freaking tiny. I have good experience with the HT7333 (250mA)for the esp8266. I will order now some HT7833 (500mA) for the esp32. In used them originally in battery powered devices because the voltage drop and energy loss is freaking low.
